RadarSDK/RadarAPIClient.m

```diff
@@ -376,6 +377,48 @@ - (void)trackWithLocation:(CLLocation *_Nonnull)location
                                          }];
     }
 
+    if (YES) {
```

Comment:
why is this `YES`?

Reply:
placeholder while I was implementing sdk config